src/raw/mod.rs

```diff
@@ -42,18 +40,34 @@ mod requests;
 /// **But, you should not need to worry about all this:** Many functions which accept a
 /// `ColumnFamily` accept an `Into<ColumnFamily>`, which means all of the above types can be passed
 /// directly to those functions.
-#[derive(Default, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
-pub struct ColumnFamily(String);
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub enum ColumnFamily {
+    Default,
+    Lock,
+    Write,
+    Version,
+}
 
 impl<T: Into<String>> From<T> for ColumnFamily {
```

Comment:
I would implement `TryFrom` rather than `From` for `String` and `&str` (not `T: Into<String>`). Then we can return an error rather than panicking if the string is an incorrect name,